spec/poros/park_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "rails_helper"
+
+RSpec.describe Park do
+  it "exists and has attributes", :vcr do
```

Comment:
No need for VCR here as there's no API call being made when calling Park.new. You should be able to remove the vcr piece and this test should still pass just fine